tests/test-theme-media.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package Create_Block_Theme
+ */
+class Test_Create_Block_Theme_Media extends WP_UnitTestCase {
+
+	public function test_make_images_block_local() {
+		$template          = new stdClass();
+		$template->content = '
+			<!-- wp:image -->
+			<figure class="wp-block-image"><img src="http://example.com/image.jpg" alt="Alternative Text" /></figure>
+			<!-- /wp:image -->
+		';
+		$new_template      = Theme_Media::make_template_images_local( $template );
+
+		// The image should be replaced with a relative URL
+		$this->assertStringNotContainsString( 'http://example.com/image.jpg', $new_template->content );
+		$this->assertStringContainsString( 'get_stylesheet_directory_uri', $new_template->content );
```

Comment:
```suggestion
		$this->assertStringContainsString( '<?php echo esc_url( get_stylesheet_directory_uri() ); ?>', $new_template->content );
```

Test for the full string, to catch encoding issues at this stage. Making this modification helped me realize that at least this encoding issue, where `<` and `>` are turned into their unicode equivalents `\u003c` and `\u0033`, is caused somewhere down the line.

Reply:
Similar test added [here](https://github.com/WordPress/create-block-theme/pull/522/commits/649ba9d952b9d83ceed5e41261951ee2051ef6cd).